Effectively disable inlining of functions which are defined
in argp-fmtstream.c because of name space conflicts when
including argp-namefrob.h

--- argp-fmtstream.h	2003-12-11 09:37:05.000000000 +0100
+++ argp-fmtstream.h	2016-09-27 09:19:48.891000000 +0200
@@ -192,6 +192,7 @@
 extern int _argp_fmtstream_ensure (argp_fmtstream_t __fs, size_t __amount);
 extern int __argp_fmtstream_ensure (argp_fmtstream_t __fs, size_t __amount);
 
+#ifdef ARGP_USE_INLINE
 #ifdef __OPTIMIZE__
 /* Inline versions of above routines.  */
 
@@ -208,7 +209,7 @@
 #endif
 
 #ifndef ARGP_FS_EI
-#define ARGP_FS_EI extern inline
+#define ARGP_FS_EI static inline
 #endif
 
 ARGP_FS_EI size_t
@@ -306,6 +307,8 @@
 
 #endif /* __OPTIMIZE__ */
 
+#endif /* ARGP_USE_INLINE */
+
 #endif /* ARGP_FMTSTREAM_USE_LINEWRAP */
 
 #endif /* argp-fmtstream.h */
